include/Song.h

```diff
@@ -257,6 +257,7 @@ class LMMS_EXPORT Song : public TrackContainer
 		return m_playMode;
 	}
 
+	inline PlayMode lastPlayMode() const { return m_lastPlayMode; }
```

Comment:
`inline` is redundant here.
```suggestion
	PlayMode lastPlayMode() const { return m_lastPlayMode; }
```